CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/foundation) [\#772](https://github.com/line/lbm-sdk/pull/772) export x/foundation pool
 * (baseapp) [\#781](https://github.com/line/lbm-sdk/pull/781) implement method `SetOption()` in baseapp
 * (global) [\#782](https://github.com/line/lbm-sdk/pull/782) add unhandled return error handling
+* (x/foundation) [\#790](https://github.com/line/lbm-sdk/pull/790) fix case of gov_mint_left_count in x/foundation
```

Comment:
Please replace this change location to `Breaking Changes`. Because this change need to be reset or migration.